src/components/OverlayWrapper/OverlayWrapper.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import _ from 'lodash';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createClass, getFirst, rejectTypes, omitProps } from '../../util/component-types';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+
+const cx = lucidClassNames.bind('&-OverlayWrapper');
+
+const {
+	bool,
+	node,
+	oneOf,
+	string,
+} = React.PropTypes;
+
+/**
+ *
+ * {"categories": ["utility"]}
+ *
+ * A wrapper with optional overlay to wrap content. `Overlay` is meant for overlaying an entire page, while this component is meant to wrap another component and cover its content.
+ *
+ */
+const OverlayWrapper = createClass({
```

Comment:
`Anchored` kind of implies that it's floating next to something and positions to it. Ideally this would just be called `Overlay` and we could rename the page level one to `PageOverlay`. But since that's breaking, I think `OverlayWrapper` is fine.